driver/driver_test.go

```diff
@@ -51,14 +51,22 @@ func TestDriver(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
+	cleanupAll()
+})
+
+var _ = AfterSuite(func() {
```

Comment:
this is how the test framework works. Notice that there is no main function anywhere. We want to run "BeforeSuite" and "AfterSuite" which are in global scope, so we need to fake assignment to variable. 

Alternatively, we could move those fuctions inside
```
var _ = Desribe(func() {
... 
}
```
but I don't like this option since we have multiple such blocks.